core/vm/jump_table.go

```diff
@@ -1030,12 +1030,12 @@ func newFrontierInstructionSet() JumpTable {
 			writes:     true,
 		},
 		ETX: {
-			execute:    opETX,
-			dynamicGas: gasCallEIP2929, // temporary
-			minStack:   minStack(10, 1),
-			maxStack:   maxStack(10, 1),
-			memorySize: memoryETX,
-			writes:     true,
+			execute:     opETX,
+			constantGas: params.TxGas, // temporary
```

Comment:
Is this still 'temporary'?

Reply:
Yes because we haven't decided what exactly the gas cost of executing the ETX opcode should be (separate from the gas inserted in the ETX)